web/adelacecilia/html/embed tag/index.html

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <title>An embedded HTML page</title>
+  </head>
+  <body>
+    <h1>Embedded HTML page</h1>
+
+    <embed
```

Comment:
Please dont leave empty lines in your HTML :)

Reply:
please delete blank line 8